content/id/docs/concepts/services-networking/dual-stack.md

```diff
@@ -0,0 +1,145 @@
+feature:
+  title: _Dual-stack_ IPv4/IPv6 
+  description: >
+    Pengalokasian alamat IPv4 dan IPv6 untuk Pod-Pod dan Layanan-layanan
+
+content_template: templates/concept
+weight: 70
+---
+
+{{% capture overview %}}
+
+{{< feature-state for_k8s_version="v1.16" state="alpha" >}}
+
+_Dual-stack_ IPv4/IPv6 memungkinkan pengalokasian alamat IPv4 dan IPv6 untuk
+{{< glossary_tooltip text="Pod" term_id="pod" >}} dan {{< glossary_tooltip text="Service" term_id="service" >}}.
+
+Jika kamu mengaktifkan jaringan _dual-stack_ IPv4/IPv6 untuk klaster Kubernetes 
+kamu, klaster akan mendukung pengalokasian kedua alamat IPv4 dan IPv6 secara 
+bersamaan.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Fitur-fitur yang didukung
+
+Mengaktifkan _dual-stack_ IPv4 / IPv6 pada klaster Kubernetes kamu untuk 
+menyediakan fitur-fitur berikut ini:
+
+   * Jaringan Pod _dual-stack_ (pengalokasian sebuah alamat IPv4 dan IPv6 
+     untuk setiap Pod)
+   * Layanan yang mendukung IPv4 dan IPv6 (setiap layanan hanya untuk satu 
+     keluarga alamat)
+   * Pod routing untuk keluar klaster (misalnya Internet) melalui antarmuka 
+     IPv4 dan IPv6
+
+## Prasyarat
+
+Prasyarat berikut diperlukan untuk menggunakan _dual-stack_ IPv4/IPv6 pada 
+klaster Kubernetes :
+
+   * Kubernetes versi 1.16 atau yang lebih baru
+   * Dukungan dari penyedia layanan akan jaringan _dual-stack_ (Penyedia layanan
+     _cloud_ atau yang lainnya harus dapat memberikan Node Kubernetes dengan 
+     antarmuka jaringan IPv4/IPv6 yang dapat dirutekan)
+   * Sebuah _plugin_ jaringan yang mendukung _dual-stack_ (seperti Kubenet atau Calico)
+   * Kube-proxy berjalan dalam mode IPVS
+
+## Mengaktifkan _dual-stack_ IPv4/IPv6
+
+Untuk mengaktifkan _dual-stack_ IPv4/IPv6, aktifkan `IPv6DualStack` [gerbang fitur](../en/docs/reference/command-line-tools-reference/feature-gates/) 
+untuk komponen yang relevan dari klaster kamu, dan atur pengalokasian jaringan 
+_dual-stack_ pada klaster:
+
+   * kube-controller-manager:
+      * `--feature-gates="IPv6DualStack=true"`
+      * `--cluster-cidr=<IPv4 CIDR>,<IPv6 CIDR>` misalnya `--cluster-cidr=10.244.0.0/16,fc00::/24`
+      * `--service-cluster-ip-range=<IPv4 CIDR>,<IPv6 CIDR>`
+      * `--node-cidr-mask-size-ipv4|--node-cidr-mask-size-ipv6` _default_ /24 
+        untuk IPv4 dan /64 untuk IPv6
+   * kubelet:
+      * `--feature-gates="IPv6DualStack=true"`
+   * kube-proxy:
+      * `--proxy-mode=ipvs`
+      * `--cluster-cidr=<IPv4 CIDR>,<IPv6 CIDR>`
+      * `--feature-gates="IPv6DualStack=true"`
+
+{{< caution >}}
+Jika kamu menentukan blok alamat IPv6 yang lebih besar dari /24 melalui 
+`--cluster-cidr` pada baris perintah, maka pengalokasian tersebut akan gagal.
+{{< /caution >}}
+
+## Layanan-layanan
+
+Jika klaster kamu mengaktifkan jaringan _dual-stack_ IPv4/IPv6, maka kamu dapat 
+membuat {{<glossary_tooltip text ="Service" term_id ="service">}} (layanan) dengan 
+alamat IPv4 atau IPv6. Kamu dapat memilih keluarga alamat untuk IP klaster  
```

Comment:
```suggestion
alamat IPv4 atau IPv6. Kamu dapat memilih keluarga alamat untuk clusterIP
```
Albeit the original English article, we can translate it to `clusterIP` to provide clarity for the reader.